### ISSUE TITLE
Outdated page:ARIA FAQ should be deleted or re-written

### ISSUE THREAD

**Issue by estelle** [mdn/content#17076]
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Web_applications_and_ARIA_FAQ

### What specific section or headline is this issue about?

General overview of ARIA

### What information was incorrect, unhelpful, or incomplete?

It was likely written in about 2008. Many dead links. Content outdated, including referencing no-longer-used frameworks.

### What did you expect to see?

useful information

**Comment by Josh-Cena** [comment]
Fixed via https://github.com/mdn/content/pull/19052
